### PR TITLE
Improve french translation

### PR DESCRIPTION
**Pull request by M2ck**

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -45,10 +45,10 @@
     <string name="rename">Renommer</string>
   <string name="save">Sauvegarder</string>
     <string name="bookmarksadded">Favoris ajoutés</string>
-  <string name="fileexist">Un fichier avec le même nom existe déjà </string>
-    <string name="name">Nom: </string>
-  <string name="date">Date: </string>
-  <string name="location">Stocké dans: </string>
+  <string name="fileexist">Un fichier avec le même nom existe déjà</string>
+    <string name="name">Nom : </string>
```
Comment (EmmanuelMess) [inline app/src/main/res/values-fr/strings.xml]: Why did you add the space here?

Comment (M2ck) [reply app/src/main/res/values-fr/strings.xml]: In french punctuation a space is required before and after a colon.
